### DISCUSSION TITLE
Processor complex emulation

### DISCUSSION THREAD

**Discussion by lazerode** [86Box/86Box#3786] [General]
Does 86Box properly support emulating machines with processor complexes? This is needed before adding some machines, in particular the IBM PS/2 model 90/95 XP.
If there is no CPU complex emulation in 86Box, do you have any idea on implementation?

**Comment by dhrdlicka** [comment]
From what I see, "processor complex" is just an IBM term for a processor card. We already do emulate certain systems that have separate processor cards (ASUS P/I-P65UP5 for one, although for technical reason each processor card configuration is listed as its own machine), so this shouldn't be a problem.

I don't think they actually need any special consideration, it's just that the system buses are wired to a slot rather than onboard chips and a socket.

**Reply by TC1995** [reply]
Well, there appears to be undocumented components for such complexes, especially the DMA one.
